src/main/java/com/danielagapov/spawn/Services/ChatMessage/ChatMessageService.java

```diff
@@ -16,39 +20,54 @@
 
 @Service
 public class ChatMessageService implements IChatMessageService {
-    private final IChatMessageRepository repository;
+    private final IChatMessageRepository chatMessageRepository;
```

Comment:
Fine for now. We might want to consider using the mediator design pattern later on to avoid all of these dependencies.